src/components/MentionTextArea/suggestion.ts

```diff
@@ -42,9 +57,9 @@ const suggestion = {
           editor: props.editor
         });
 
-        spinner = tippy('body', {
+        [spinner] = tippy('body', {
           getReferenceClientRect: getClientRect(props),
-          appendTo: () => document.getElementById(editorID) || document.body,
+          appendTo: props.editor.options.element,
```

Comment:
Updated this to use the editor element instead of getting it from the `id` prop. This way we don't have to have duplicate `id` attributes on both the editor and container.